src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1022,7 +1022,7 @@
                   </widget>
                  </item>
                  <item>
-                  <widget class="QLabel" name="labelSmartFee2">
+                  <widget class="QLabel" name="labelPriority2">
                    <property name="text">
                     <string>(Smart fee not initialized yet. This usually takes a few blocks...)</string>
```

Comment:
Nit: Do we need to change this string? I haven't been able to trigger it to display.

Reply:
Removed it entirely